indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -141,9 +141,19 @@ static RealtimeTuningConfig convertTuningConfig(ShardSpec spec, IndexTuningConfi
 
   private final ObjectMapper jsonMapper;
 
+  public IndexTask(
```

Comment:
Hmm, apparently so - I did a search for implementers and thought I had hits, but it was late. It is, indeed, only tests that directly instantiate IndexTask. I can update the implementations in IndexTaskTest, TaskSerdeTest, and TaskLifecycleTest and eliminate the constr overload.